libraries/launcher/org/prismlauncher/log4jinjector/UniversalLog4jInjector.java

```diff
@@ -0,0 +1,38 @@
+package org.prismlauncher.log4jinjector;
+
+import org.prismlauncher.utils.logging.Log;
+
+import java.util.Objects;
+
+public class UniversalLog4jInjector {
+    private static final String EXCEPTION_DURING_INJECTION = "Unable to inject into Log4j. Token may be written to a file as a result";
```

Comment:
Just a *tiny* suggestion.
```suggestion
    private static final String EXCEPTION_DURING_INJECTION = "Unable to inject into Log4j. Token may be written to a file as a result.";
```